server/db/dbcontrollers/stocksController.js

```diff
@@ -25,5 +25,53 @@ module.exports = function (knex) {
       .orWhere(knex.raw('UPPER(name) like UPPER(?)', [searchLike]));
   };
 
+  //pass in array of objects contaning symbol,shares,and price 
+  //query stock_prices and update price if needed and return
+  module.getPrices = function (stockArr) {
+    
+    return knex.select('symbol','ask')
+      .from('stock_prices')
+    .then(function (dataArr) {
+
```

Comment:
This seems like it could be really expensive. It's pulling all prices from stock prices and then looping through them to find each stock I own. That's a lot of looping on an array with 6000 stocks.

I don't think this stops anything from working, but we can probably change the order of operations to make this work a little more efficiently.